### PR TITLE
v6.4.0 - Apply revision identifier to CSS filename

### LINKED ISSUE #99: minified css file name - always hash value appended
Hi All,

I'm not sure if this is a bug or whether I'm doing something wrong but when I try to do a production build I'm always getting a hash value at the end of the minified css file instead of style.min.css.

I've noticed js has an option `applyRevision`

But the CSS doesn't and always applies a revision hash number to the end, is there a way to disable this?

I see in the tasks for scripts there is 
```
.pipe(gulpif(applyRevision,
	rev()
))
```
But in the CSS tasks it always revisions
```
.pipe(rev())
```

Thanks,
Tom

### PR DESCRIPTION
**Pull request by xander-marjoram**
Moved the applyRevision variable out of the JS-only config, also updated readme and unit tests.

Fixes #99.

### CODE REVIEW

File: tasks/css.js [hunk 1]
```diff
@@ -106,6 +106,8 @@ gulp.task('css:lint', () => gulp.src(`${pathBuilder.cssDistDir}/**/*.css`)
  */
 gulp.task('css:bundle', () => {
 
+    const { applyRevision } = config;
```
Comment (DamianMullins) [inline tasks/css.js]: Looking through the rest of the file I think you should just call `config.applyRevision` below, keeping the code style consistent throughout the project.

File: test/config.test.js [hunk 2]
```diff
@@ -182,22 +197,6 @@ describe('javascript config', () => {
...
-        // Arrange
-        const applyRevision = false;
-
-        // Act
-        config.update({ js: { files: { main: { applyRevision } } } });
-
-        // Assert
-        expect(config.js.files.main.applyRevision).toBe(applyRevision);
-    });
-
```
Comment (DamianMullins) [inline test/config.test.js]: NIT: add this space back in.

File: tasks/css.js (hunk 1 above)
Comment (xander-marjoram) [reply tasks/css.js]: @DamianMullins Should I change this [here](https://github.com/justeat/gulp-build-fozzie/blob/eacb94812bf7d30bf24b6904340c1de35429115d/tasks/javascript.js#L166) too?

Comment (DamianMullins) [reply tasks/css.js]: Yeah, change in both places.

**Comment by ashleynolan** [comment]
@xander-marjoram Can you make sure you publish this up to NPM now it's merged? 🚀

**Pull request merged**